src/alg_utils.jl

```diff
@@ -565,6 +569,9 @@ alg_order(alg::MSRK5) = 5
 alg_order(alg::MSRK6) = 6
 alg_order(alg::Stepanov5) = 5
 alg_order(alg::SIR54) = 5
+alg_order(alg::PSRK4p7q6) = 4
+alg_order(alg::PSRK3p6q5) = 3
+alg_order(alg::PSRK3p5q4) = 3
```

Comment:
Can we also get some tests for these, please? See my suggestion for test/algconvergence/ode_convergence_tests.jl below

Reply:
I made the test locally considering 

`f = (u, p, t) -> sin(u)
prob_ode_nonlinear = ODEProblem(
    ODEFunction(f;
        analytic = (u0, p, t) -> 2 * acot(exp(-t) *
                                          cot(0.5))), 1.0,
    (0.0, 0.5))`
    
    with the following results:
    `sim1.𝒪est[:l∞] = 4.001743674308167
sim2.𝒪est[:l∞] = 3.0568000432504037
sim3.𝒪est[:l∞] = 2.9555769869783797
`

With the problem defined in the ode_convergence file for low order rk schemes the results are all 4th order accuracy, rather than being 3rd order. Should I add this non_linear problem in the same file where I added the test for these schemes or should I move them in another file (maybe where this non linear problem is already available) without defining a new prob?

ps: I also checked the solution with some other test cases by plotting the solutions and everything looks fine.

Thank you for your time!

Reply:
Please add the new tests to the same file where you added the test for these schemes